app/templates/components/ui-table/cell/cell-sponsor-sanitize.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if (not-eq record.type 'null')}}
+  <span>{{record.type}}</span>
```

Comment:
Instead of hard-coding a property within the template, use the propertyName to retrieve it.

With something like (get record column.propertyName) in the template